.meteor/packages

```diff
@@ -96,3 +96,4 @@ johanbrook:publication-collector
 
 
 # Custom Packages
+natestrauser:select2
```

Comment:
@zenweasel I used a Meteor package because it was under the `Usage` section of the above NPM package.